build/static/style.css

```diff
@@ -0,0 +1,21 @@
+.workload-summary-cell{
+  max-width: 40px;
+  overflow: hidden;
+}
+
+.red, .green{
+  color: white;
+}
+.red{
+  background-color: red;
+}
+.green{
+  background-color: green;
+}
+.yellow{
+  background-color: yellow;
+}
+
+table tr td{
```

Comment:
why is this such a specific selector? 
`td`s should never be used outside a table or a table row.

use
```
td {

...
}
```

Reply:
Right, that was a weird choice. Simplified.